CHANGELOG.md

```diff
@@ -7,8 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for multiline string literals within compiler directives.
+- Support for the `TEXTBLOCK` directive.
+- **API:** `CompilerDirectiveParser` can now return a new `TextBlockDirective` type.
+- **API:** `CheckVerifier::withCompilerVersion` method.
+- **API:** `CheckVerifier::withToolchain` method.
+
 ### Changed
 
+- `NativeInt` and `NativeUInt` are now treated as weak aliases in Delphi 12+.
+- The length of open arrays is now modeled as `NativeInt` in Delphi 12+.
```

Comment:
The length of open arrays is now modeled as `NativeInt` regardless of Delphi version. If this is unintentional, it should be fixed elsewhere, but if it's intended behaviour then this changelog entry should be changed.

Reply:
Unintended behavior, the changelog is correct.